wiki/Ranking_criteria/osu!catch/en.md

```diff
@@ -153,14 +153,15 @@ Difficulty-specific rules and guidelines do only apply to the difficulty level t
 - **[Hyperdashes](/wiki/Gameplay/Hyperdash) and [dashes](/wiki/Gameplay/Dash) must have at least a 62 ms gap between their two objects.**
 - **[Hyperdashes](/wiki/Gameplay/Hyperdash) that are [basic-snapped](/wiki/Gameplay/Dash_snapping#basic-snapped) must not be used more than four times between consecutive [fruits](/wiki/Gameplay/Hit_object/Fruit).**
 - **[Hyperdashes](/wiki/Gameplay/Hyperdash) that are [basic-snapped](/wiki/Gameplay/Dash_snapping#basic-snapped) must not be used more than two times within a slider.** The slider path must be simple and easy-to-follow.
-- **[Hyperdashes](/wiki/Gameplay/Hyperdash) that are [higher-snapped](/wiki/Gameplay/Dash_snapping#higher-snapped) must not be used in conjunction with higher-snapped [dashes](/wiki/Gameplay/Dash) or any other hyperdashes.**
+- **[Hyperdashes](/wiki/Gameplay/Hyperdash) that are [higher-snapped](/wiki/Gameplay/Dash_snapping#higher-snapped) must not be used in conjunction with any other hyperdashes.**
```

Comment:
i don't fully understand it either. may need a catch mapper to explain it better

Reply:
for sure, I also left a comment on the RC draft where this will come up. it seems like overall they are planning to improve terminology regarding classifying different types of dashes.

for this PR, I don't think anything needs to change though